update.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I'm a little unclear on the intended use of `update.sh`; I think it's meant to be used by developers when making a change to a configuration file, making it easier to distribute that to each of the variants. Is that right?

Reply:
Yes this is correct. It can also be used by a bot (like @docker-library-bot) to update the images.